pkg/kobject/kobject.go

```diff
@@ -67,7 +67,7 @@ type ServiceConfig struct {
 	Command         []string            `compose:"command" bundle:"Command"`
 	WorkingDir      string              `compose:"" bundle:"WorkingDir"`
 	Args            []string            `compose:"args" bundle:"Args"`
-	Volumes         []string            `compose:"volumes" bundle:"Volumes"`
+	VolList         []string            `compose:"volumes" bundle:"Volumes"`
```

Comment:
what is a difference between `VolList` and `Volumes`? What is purpose of `VolList`? We need comments describing both